test_runner/fixtures/neon_fixtures.py

```diff
@@ -383,8 +387,8 @@ class AuthKeys:
     pub: bytes
     priv: bytes
 
-    def generate_management_token(self):
-        token = jwt.encode({"scope": "pageserverapi"}, self.priv, algorithm="RS256")
+    def generate_management_token(self) -> str:
+        token = jwt.encode({"scope": "pageserverapi"}, cast(str, self.priv), algorithm="RS256")
```

Comment:
This part looks a bit suspicious. Shouldn't we define `pub` and `priv` as strings rather than bytes?

Reply:
good catch, `encode`/`decode` accepts `key` as a `str`

Reply:
how can I run tests that require such tokens to check it? Because I've also changed the reading from bytes to text

Reply:
We use `auth_keys` explicitly in:
```
test_runner/regress/test_auth.py
test_runner/regress/test_pageserver_api.py
test_runner/regress/test_wal_acceptor.py
```

On CI all the tests are run.